static/js/stream_edit.js

```diff
@@ -552,8 +552,15 @@ exports.initialize = function () {
 
         function invite_failure(xhr) {
             const error = JSON.parse(xhr.responseText);
-            stream_subscription_info_elem.text(error.msg)
-                .addClass("text-error").removeClass("text-success");
+            // The message has been changed to "Unable to find user" but not in the API
+            if (error.msg.slice(0, 51) === "User not authorized to execute queries on behalf of") {
```

Comment:
I just noticed [this commit](https://github.com/zulip/zulip/commit/7d6ee2e10aec95b2978306d8f1acb6f7c4244be7).  It looks like we already had this behavior, and then switched to displaying the backend error.  It's probably worth it to bring this up in chat, and inquire how to proceed with this one.